chore: update readme (#56)
cookies-mod/mod: README.md

Mainly used to toggle certain debug functionality, disabling of checks, extra logging and printing debug information

## /cookie
- Also available as `/cookies:cookie`

### sendCoords

> > `/cookie sendCoords` sends your current coordinates as chat message e.g. `x: -10, y: 20, z: 10`

Reviewer: For the sake of clarity, it should be clarified whether this chat message is only visible to you or if it's an actual chat message that is sent to all players.

If it's only visible for the client, I'd suggest re-phrasing the message as
```suggestion
> `/cookie sendCoords` displays your current coordinates for you in the chat. Example: `x: -10, y: 20, z: 10`
```

Author: The message is actually sent in chat, so the original phrasing should be correct.
